packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -207,6 +207,10 @@
     font-size: 14px;
     font-family: "Mafs-MJXTEX";
     line-height: 1.5em;
+    /* Prevent labels from being selected, and from interfering with click and
+     * drag interactions on the graph. */
+    user-select: none;
+    pointer-events: none;
```

Comment:
What is the result of `pointer-events: none` here? I'm not sure it changes any behavior, because afaik it don't affect drag selection? Or is it just here for future proofing?

Reply:
It prevents the axis tick labels from interfering with drag and hover interactions on the graph.